components/scream/src/physics/shoc/tests/shoc_unit_tests_common.hpp

```diff
@@ -113,6 +113,7 @@ struct UnitWrap {
     struct TestDiagSecondShocMoments;
     struct TestPblintdCldCheck;
     struct TestComputeShocVapor;
+    struct TestUpdatePrognosticsImplici;
```

Comment:
I think `t` is missing in this `struct` name. It is missing everywhere, so it is consistent. It doesn't matter code wise but it will be good to be consistent with the long name of the function.

Reply:
@singhbalwinder , thanks, I will fix.